jupyter_core/command.py

```diff
@@ -57,8 +58,8 @@ def list_subcommands():
         except OSError:
             continue
         for name in names:
-            if name.startswith('jupyter-'):
-                subcommands.add(name.split('-')[1])
```

Comment:
`.split()` also takes an optional second parameter for the maximum number of times to split, so `name.split('-', 1)[1]` would also work correctly. I don't know that this is any neater, though.